packages/terra-application-links/src/tabs/ApplicationTabs.module.scss

```
@@ -42,123 +43,180 @@
+      background-color: var(--terra-application-links-tabs-tab-background-color, transparent);
+      background-position: var(--terra-application-links-tabs-tab-background-position, top left);
+      background-repeat: no-repeat;
+      background-size: var(--terra-application-links-tabs-tab-background-size, 0 5px);
+      box-shadow: var(--terra-application-links-tabs-tab-box-shadow);
+      box-sizing: border-box;
+      display: flex;
+      flex: 0 0 auto;
+      height: 100%;
+      justify-content: center;
+      left: var(--terra-application-links-tabs-tab-position-left, 2px);
+      overflow: hidden;
+      padding-left: var(--terra-application-links-tabs-tab-padding-left, 1.0715rem);
+      padding-right: var(--terra-application-links-tabs-tab-padding-right, 1.0715rem);
+      position: absolute;
+      top: 0;
+      transition-duration: var(--terra-application-links-tabs-tab-transition-duration, 0s);
+      transition-property: background-size;
+      transition-timing-function: var(--terra-application-links-tabs-tab-transition-timing-function, ease);
+      width: calc(100% - var(--terra-application-links-tabs-tab-divider-before-width, 2px));
+    }
+
+    .tab-label {
       display: block;
+      max-width: 100%;
       overflow: hidden;
       position: relative; // remove IE10 active click transformation
       text-overflow: ellipsis;
       white-space: nowrap;
-      width: 100%;
       word-break: normal;
       word-wrap: normal;
     }
 
     &:hover,
     &.is-hovered {
-      background-color: var(--terra-application-tabs-hover-background-color, rgba(0, 0, 0, 0.15));
+      .tab-inner {
+        background-color: var(--terra-application-links-tabs-tab-hover-background-color, rgba(0, 0, 0, 0.15));
+      }
     }
 
     &[aria-current='true'] {
-      background-color: var(--terra-application-tabs-hover-background-color, rgba(0, 0, 0, 0.15));
-      background-image: var(--terra-application-tabs-selected-background-image, linear-gradient(to bottom, #5cc2f9, #5cc2f9));
-      background-size: var(--terra-application-tabs-active-background-size, 13rem 5px);
-      font-weight: 600;
+      font-weight: var(--terra-application-links-tabs-tab-selected-font-weight, 500);
+      text-shadow: var(--terra-application-links-tabs-tab-selected-text-shadow, none);
+
+      &::before {
+        background-color: var(--terra-application-links-tabs-tab-selected-divider-before-background-color, #0079be);
+        background-image: var(--terra-application-links-tabs-tab-selected-divider-before-background-image);
+        border-left-color: var(--terra-application-links-tabs-tab-selected-divider-before-border-left-color, rgba(255, 255, 255, 0.17));
+        border-right-color: var(--terra-application-links-tabs-tab-selected-divider-before-border-right-color, rgba(0, 0, 0, 0.17));
+      }
+
+      .tab-inner {
+        background-color: var(--terra-application-links-tabs-selected-background-color, rgba(0, 0, 0, 0.15));
```

we should add `tab` to be consistent with the above blocks and separate `.tab-inner` block on line 119
```suggestion
        background-color: var(--terra-application-links-tabs-tab-selected-background-color, rgba(0, 0, 0, 0.15));
```

---

Should be more consistent here
https://github.com/cerner/terra-framework/pull/439/commits/fbd5fa6578ceca17b1778f3df76dec9e33c387a4#diff-c35c5a53e93ecb7b1096d5e08a498c61R171
https://github.com/cerner/terra-framework/pull/439/commits/fbd5fa6578ceca17b1778f3df76dec9e33c387a4#diff-c35c5a53e93ecb7b1096d5e08a498c61R294

---

Looks good to me